game/src/core/utils/components/position/Direction.java

```diff
@@ -0,0 +1,9 @@
+package core.utils.components.position;
+
+/** Represents the possible directions an entity can face or move in. */
+public enum Direction {
```

Comment:
Vlt lieber als inner Enum in `PositionComponent`. Wir haben sonst insgesamt 4x `Direction` 😅.
Sonst müsste man das mal später refactoren das man nur eins in Game hat?

Reply:
Haben wir ein Direction mit UP;DOWN;LEFt;RIGHT? Dann nehmen wir einfach das. Ich hatte keins gefunden